test/fixtures/elm/queries/SimpleQueriesDependency.cql

```diff
@@ -1,11 +0,0 @@
-library SimpleDep version '0.0.1'
```

Comment:
was this an intentional nuking of this file?

Reply:
nope. Brought it back.